public/app/features/variables/pickers/PickerRenderer.tsx

```diff
@@ -48,25 +48,17 @@ function PickerLabel({ variable }: PropsWithChildren<Props>): ReactElement | nul
       return {};
     }
 
-    /**
-     * TODO:
-     * The below ts-ignores result from the fact that we expect MuiTheme in Grafana
-     * Let's pass GrafanaTheme
-     */
-    // @ts-ignore
-    const color = theme?.palette?.text?.primary;
-    // @ts-ignore
-    const lightBackgroundColor = theme?.palette?.grey;
-    // @ts-ignore
-    const darkBackgroundColor = theme?.palette?.background?.default;
-
     const commonStyles: CSSProperties = {
       ...COMMON_PICKER_LABEL_STYLE,
-      ...(color ? { color } : {}),
+      color: theme.palette.text.primary,
```

Comment:
LEt's override grafana color. Do not use muiTheme

Reply:
Grafana theme is already using our colors I have changed initial theme in theme package before you joined Fluxninja. If you run grafana in localhost:3003 and compare it with actual Grafana you will know. This theme is passed just because we want to use it here. Theme override is completely waste effort.

Reply:
THere should no be need to use the theme here.